pkg/java/pom/parse.go

```diff
@@ -291,17 +296,17 @@ func (p parser) mergeDependencyManagement(a, b map[string]pomDependency) map[str
 	return a
 }
 
-func (p parser) parseDependencies(deps []pomDependency, props map[string]string, depManagement map[string]pomDependency,
+func (p parser) parseDependencies(deps *pom, props map[string]string, depManagement map[string]pomDependency,
 	exclusions map[string]struct{}) []artifact {
 	var dependencies []artifact
-	for _, d := range deps {
+	for _, d := range deps.content.Dependencies.Dependency {
```

Comment:
Could you clarify the reason for this change?

Reply:
you are right. It is extra changes. I revert it.